### ISSUE TITLE
Usability improvements for zoom function

### ISSUE THREAD

**Issue by clavinet** [SimpleMobileTools/Simple-Gallery#1982]
I installed the latest version from F-Droid, and made some comparisons with the stock AOSP/LineageOS gallery.

I found that your app is much faster, but there are some usability issues imo with zooming.

1. When flipping through images, pinch-zooming only works after the flip animation has fully completed, AOSP gallery lets you zoom basically always.
2. Similarly, when starting a flip with one finger, then later doing a pinch gesture using another finger, Simple Gallery doesn't recognize it as zoom, but AOSP gallery will infer that by using two fingers, you want to zoom.
3. When the image is zoomed in, you can't flip to another one, you have to zoom out to be able to flip.

To sum it up, AOSP gallery seems "smarter" about when and how to zoom.

What do you think?

**Comment by tibbi** [comment]
I dont really like those, some zooming related limitations have been added intentionally to avoid glitching and unwanted behaviour. It seems to work just fine to people.
